### DISCUSSION TITLE
how can use explorer open current buffer

### DISCUSSION THREAD

**Discussion by 1925381584** [helix-editor/helix#11335] [Q&A] [answered]
hi, how can use explorer open current buffer? I was going to use this command, but I don't know how to get the path of the current buffer. Is such an operation supported now?

<img width="200" alt="IPBt7FRNOH" src="https://github.com/user-attachments/assets/23c671db-ebf9-4894-9d1f-edfe163bdd06">

**Comment by kirawi** [comment, marked as answer]
`Ctrl-r` `%` with the prompt open

**Comment by 1925381584** [comment]
thanks

**Reply by kirawi** [reply]
Not yet, it requires https://github.com/helix-editor/helix/pull/11164 or https://github.com/helix-editor/helix/pull/4709

**Comment by 1925381584** [comment]
but can it only be viewed interactively? Can't the steps be bound to a shortcut?
